pkg/controller/import-controller.go

```diff
@@ -699,6 +723,42 @@ func getEndpoint(pvc *corev1.PersistentVolumeClaim) (string, error) {
 	return ep, nil
 }
 
+// returns the OpenShift cluster wide proxy object
+func getClusterWideProxy(log logr.Logger, client client.Client) (*ocpconfigv1.Proxy, error) {
```

Comment:
does it work? looks strange, where is the return when client.Get returns without error?

Maybe this function deserves a simple unit test

Reply:
Thanks for points this, I fixed the code and created the unit tests.